src/logger.ts

```diff
@@ -27,13 +28,16 @@ export const LoggerLevel = Object.freeze({
   debug: 'debug'
 } as const);
 
-/** @public */
+/** @public
+ * @deprecated The Legacy Logger is deprecated and will be removed in the next major version.*/
```

Comment:
```suggestion
/** 
 * @public
 * @deprecated The Legacy Logger is deprecated and will be removed in the next major version.
 */
```

Reply:
I won't comment on them all but the public tag does not get picked up if its on the same line as the `**`

Reply:
The comments have to either be all one line, or with a line break after the `**`

Reply:
Good catch